include/fiction/utils/layout_utils.hpp

```diff
@@ -350,7 +351,56 @@ Lyt convert_to_fiction_coordinates(const sidb_cell_clk_lyt_siqad& lyt) noexcept
 
     return lyt_new;
 }
+/**
+ * Generates a random cell between two given cells.
```

Comment:
What does "between" really mean here? Could you specify the functionality more elaborately in the docstring, please?